### PR TITLE
CLI/Import - optimized csv_to_dataset() method

### LINKED ISSUE #2803: CLI - Import.csv_to_dataset() inproperply parses multiline values from CSV
`robottelo.cli.import_.Import.csv_to_dataset()` method produces wrong output while parsing a CSV file with multiline values (containing `\r`). It treats each new line as a separate record.

Example:
the following CSV file contains a single (multiline) record:
```csv
server_id,profile_name,hostname,description,organization_id,organization_name,base_channel_id,base_channel_label,child_channel_id,child_channel_label,system_group_id,system_group,release,architecture,virtual_host,virtual_guest,is_virtualized,virt_type
1,profile1,tyan-gt24,"Initial Registration Parameters:
OS: redhat-release-server
Release: 7Server
CPU Arch: x86_64",1,Satellite QE,103,rhel-x86_64-server-7,None;106,None;rhn-tools-rhel-x86_64-server-7,,,7Server,x86_64,,,No,
```
the output of `csv_to_dataset` is:
```python
>>> for i in profiles:
...     i
... 
{u'server_id': u'1', u'profile_name': u'profile1', u'hostname': u'tyan-gt24', u'description': u'"Initial Registration Parameters:\r'}
{u'server_id': u'OS: redhat-release-server\r'}
{u'server_id': u'Release: 7Server\r'}
{u'organization_name': u'None;106', u'server_id': u'CPU Arch: x86_64"', u'system_group': u'', u'description': u'103', u'child_channel_label': u'7Server', u'base_channel_label': u'', u'hostname': u'Satellite QE', u'child_channel_id': u'', u'organization_id': u'rhel-x86_64-server-7', u'profile_name': u'1', u'architecture': u'No', u'system_group_id': u'x86_64', u'release': u'', u'virtual_host': u'', u'base_channel_id': u'None;rhn-tools-rhel-x86_64-server-7'}

```

the expected behavior would be:
```python
>>> for i in profiles:
...     i
... 
{u'organization_name': u'Satellite QE', u'server_id': u'1', u'system_group': u'', u'is_virtualized': u'No', u'description': u'"Initial Registration Parameters:\rOS: redhat-release-server\rRelease: 7Server\rCPU Arch: x86_64"', u'child_channel_label': u'None;rhn-tools-rhel-x86_64-server-7', u'virt_type': u'', u'base_channel_label': u'rhel-x86_64-server-7', u'hostname': u'tyan-gt24', u'child_channel_id': u'None;106', u'organization_id': u'1', u'virtual_guest': u'', u'profile_name': u'profile1', u'architecture': u'x86_64', u'system_group_id': u'', u'release': u'7Server', u'virtual_host': u'', u'base_channel_id': u'103'}
```

Need to implement catching the `\r` substring.

### PR DESCRIPTION
**Pull request by rplevka**
Modify method `csv_to_dataset` - the method now uses native csv module
  and correctly parses multiline records (Fixes #2803)
It also properly downloads the csv file to parse it instead of abusing the `ssh.command('cat {}')` command.

### CODE REVIEW

File: robottelo/cli/import_.py [hunk 1]
```diff
@@ -405,6 +407,31 @@ def content_view_with_tr_data(cls, options=None):
         return (result, transition_data)
 
     @classmethod
+    def content_host_with_tr_data(cls, options=None):
+        """Import Content Hosts (from spacewalk-report system-profiles).
+
+        :returns:
```
Comment (oshtaier) [inline robottelo/cli/import_.py]: ?

File: robottelo/cli/import_.py [hunk 2]
```diff
@@ -405,6 +407,31 @@ def content_view_with_tr_data(cls, options=None):
...
+        :raises:
```
Comment (oshtaier) [inline robottelo/cli/import_.py]: Does it rise something?

Comment (rplevka) [inline robottelo/cli/import_.py]: Bah...this piece of code is not supposed to be here (it's supposed for another commit)..let me delete that

**Comment by oshtaier** [comment]
Title of that PR doesn't help me understand in which way it is updated :)
It seems that you moved to temp file idea
ACK

**Comment by sthirugn** [comment]
ACK

**Pull request merged**